assets/messages.yml

```
@@ -88,6 +88,17 @@ messages:
 
         ~This issue has been temporarily closed as {color:#d04437}*Awaiting Response*{color}. Once the requested information has been delivered, the report will be reopened automatically.~
 
+        %quick_links%
+      fillname: []
+  attach-debug-screen-image:
+    - project: mc
+      name: Attach F3 debug screenshot
+      shortcut: fscreen
+      message: |-
+        Please attach a screenshot of this occurring with the F3 debug screen enabled.
```

Maybe using "while" here makes it a little bit unambiguous, otherwise one might (though it is rather unlikely) think that the bug occurs in combination with the debug screen.
```suggestion
        Please attach a screenshot of this occurring while the F3 debug screen is enabled.
```